src/Adapter/MSTest.CoreAdapter/Resources/Resource.resx

```diff
@@ -313,4 +313,7 @@
   <data name="TestAssembly_AssemblyDiscoveryFailure" xml:space="preserve">
     <value>Failed to discover tests from assembly {0}. Reason:{1}</value>
   </data>
+  <data name="UTA_TestMethodExpectedParameters" xml:space="preserve">
```

Comment:
Xlf file changes?
@pvlakshm Please review the string

Reply:
Done.